pkg/microservice/aslan/core/environment/handler/environment.go

```diff
@@ -389,38 +389,25 @@ func ListGroups(c *gin.Context) {
 	c.Writer.Header().Set("X-Total", strconv.Itoa(count))
 }
 
-func ListK8sWorkLoads(c *gin.Context) {
+type ListWorkloadsArgs struct {
+	Namespace    string `json:"namespace" form:"namespace"`
+	ClusterID    string `json:"clusterId" form:"clusterId"`
+	WorkloadName string `json:"workloadName" form:"workloadName"`
```

Comment:
you can use the same filter logic as before

Reply:
leave it tomorrow